### DISCUSSION TITLE
[Projects Beta] Unable to add new iteration before existing one

### DISCUSSION THREAD

**Discussion by SpencerKaiser** [community/community#32136] [Projects and Issues]
If you create an iteration in the future (next week, for example) a break is automatically set for the time from `now` up until when that iteration starts.

At that point in time, it becomes impossible to create an iteration for time between `now` and when that iteration starts... if you try to make a new iteration, the dates for the current week are greyed out and trying to manually set them shows a validation error saying the start date must before the start date of the next planned iteration

I want one of the below:
- Option to convert a `Break` into an iteration
- Option to manually set the date of the new iteration to be before the existing one

**Comment by Potherca** [comment]
Although the "break" feature is great, we have a usecase that still needs an "in between" iteration.
We need to abandon the current sprint because of a shift in the short-term priorities.
Everthing else remains the same (issues assigned to sprints, etc.) we just need an iteration in between two existing ones (where there is now a "break").

Either being able to assign issues to that break or being able to add an interation in between two existing ones (after changing the dates, so there is no overlap) would be of great help!

Also related to: https://github.com/community/community/discussions/6494

**Comment by stanlrt** [comment]
This makes no sense, the UI doesn't even convey that. Terrible UX.

**Comment by WMahoney09** [comment]
Same case for me; **Priorities shifted** and I have planned more than 10 iterations out, all of which still need to happen, but just two weeks later than originally planned to make room for the new priorities. Currently, it seems my options are:
1. manually move 50+ issues down an iteration _(do-able but painful, and I'd rather be working on completing tickets than working on arranging them)_
2. use the "add break" feature, and manage these tickets somewhere else... in the repo I guess rather than the project?

What I'd love to have:
1. **Be able to re-order iterations** _(I recognize this is potentially a big ask, but I think it would solve this use case and handle managing shifting priorities more broadly -- it would be really useful 😄 )_
3. **"insert iteration" same as "insert break"** _(would not allow for re-ordering two existing iterations, but seeing "breaks" bump later iterations dates down the calendar as needed, seems like this might be the quickest to implement?)_
4. Have an **API/CLI so I can bulk edit** issue iterations -- _I used the Issues REST API for duplicating a repo's issues to a new repo and it worked fairly well, but I could not find any documentation on setting project or project related fields in the [docs](https://docs.github.com/en/rest/issues?apiVersion=2022-11-28#list-issues-for-a-repository) -- any direction here would be very welcome!_

I find a ton of value in the new Projects feature, I use it personally and professionally -- thank you for building it! 🛠️

**Reply by brettwillis** [reply]
Well said, 💯 agree. We plan iterations in advance and often need to reprioritise which is very painful.

**Comment by jan-matejka** [comment]
This is also problematic when migrating from different system to github projects as you can't just start using github projects and then fill out the past iterations and migrate old issues to using the field.

**Comment by josh-berry** [comment]
I also want to re-order iterations. Priorities change, and not having the ability to shuffle iterations around easily makes me rather reluctant to plan more than 1-2 iterations ahead, which isn't really adequate when you have a backlog with hundreds of issues.
